templates/volunteer.html

```diff
@@ -0,0 +1,51 @@
+{% extends "page.html" %}
+{% block content %}
+<div id="attend-content" class="page-content">
+    <div id="hero">
+        <div class="hero-wrapper">
+            <div class="hero-text-wrapper">
+                <div class="hero-text">
+                    <h1>Volunteer for the Conference</h1>
+                    <p>To keep the ticket price accessible for everyone and provide the best experience as possible, we
+                        need volunteers to lend a hand!</p>
+                    <div class="cta-row">
+                        <a href="mailto:conference@foundation.matrix.org?subject=Volunteering for the Conference"
+                            class="cta">Contact us</a>
+                    </div>
+                </div>
+            </div>
+            <div class="pattern-wrapper"></div>
+        </div>
+    </div>
+    <div id="getting-there">
+        <div class="content-wrapper">
+            <div class="content">
+                <div class="col">
+                    <p>
+                        We are looking for help in those areas:
```

Comment:
```suggestion
                        We are looking for help in these areas:
```
re: the msc t-shirt discussion!